cdc/tests/utils/rawkv_data/checksum.go

```diff
@@ -0,0 +1,82 @@
+// Copyright 2020 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/pingcap/kvproto/pkg/kvrpcpb"
+	"github.com/pingcap/log"
+	"github.com/spf13/cobra"
+	"github.com/tikv/client-go/v2/rawkv"
+)
+
+func NewChecksumCommand() *cobra.Command {
+	return &cobra.Command{
+		Use:          "checksum",
+		Short:        "Verify that the upstream and downstream data are consistent",
+		SilenceUsage: true,
+		Args:         cobra.NoArgs,
+		RunE: func(cmd *cobra.Command, _ []string) error {
+			return runChecksum(cmd)
+		},
+	}
+}
+
+func runChecksum(cmd *cobra.Command) error {
+	cfg := &Config{}
+	err := cfg.ParseFromFlags(cmd.Flags())
+	if err != nil {
+		return err
+	}
+	ctx := context.Background()
+
+	if cfg.DstPD == "" {
+		return fmt.Errorf("Downstream cluster PD is not set")
+	}
+	if cfg.DstPD == cfg.SrcPD {
```

Comment:
Suggest to check cluster id (`rawkv.Client.ClusterID()`). The different PD endpoints may be from the same cluster.